.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Stale'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@main
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-pr-message: 'Hello there,
+          We notice that this PR has been inactive for awhile now. It would be a waste of your time and effort if we just close this PR and
+          throw away the SVGs in here. Thus, 1 month from now, if this PR remains inactive/stale, we will fork your repo, fix any issues,
```

Comment:
I know this text is from our existing `stale` comments and in theory this would be great, but in fact this is not done with every pull request. Maybe we could adjust the formulation?

Reply:
What changes should be made in the message? 🙂

Reply:
The message has been changed. Thanks! 🙂